packages/blockchain/package.json

```diff
@@ -26,8 +26,9 @@
     "tsc": "../../config/cli/ts-compile.sh",
     "lint": "../../config/cli/lint.sh",
     "lint:fix": "../../config/cli/lint-fix.sh",
+    "tape": "tape -r ts-node/register",
     "test": "npm run test:node && npm run test:browser",
-    "test:node": "tape -r ts-node/register ./test/*.spec.ts",
```

Comment:
I never really got when this `ts-node/register` is necessary and when it is not, can someone explain this to me?

Reply:
Is [this](https://github.com/substack/tape#preloading-modules) what you mean?  If I'm reading this correctly, we're telling `tape` to `require('ts-node/register') which then ensures that `ts-node` will compile all `.ts` files and then pass them to tape to run the tests using source maps so that stack traces will be legible when looking at test output.

I guess in this case, it was just telling tape to run all `.ts` files through ts-node before running the test scripts.

Reply:
Ok, and why we don't need this any more here?

Reply:
I replaced `tape -r ts-node/register` with `npm run tape --` so the `test:node` script uses the `tape` script.

Reply:
Ah, sorry for this likely confusing re-questioning. I actually misread the whole thing. 😜